src/doc/trpl/iterators.md

```diff
@@ -86,16 +85,17 @@ see it. This code works fine too:
 ```rust
 let nums = vec![1, 2, 3];
 
-for num in nums.iter() {
+for num in &nums {
     println!("{}", *num);
 }
 ```
 
-Now we're explicitly dereferencing `num`. Why does `iter()` give us references?
-Well, if it gave us the data itself, we would have to be its owner, which would
-involve making a copy of the data and giving us the copy. With references,
-we're just borrowing a reference to the data, and so it's just passing
-a reference, without needing to do the copy.
+Now we're explicitly dereferencing `num`. Why does `&nums` give us
+references?  Because we asked it to with `&`. If we had not had the
+`&`, `nums` would have been moved into the `for` loop and consumed,
```

Comment:
well, it wouldn't have, we'd have gotten an error, right? Or does `num`s turn into `into_iter`?